certified-connectors/Adobe Creative Cloud Libraries/README.md

```diff
@@ -0,0 +1,2 @@
+# Adobe Creative Cloud Libraries
+Adobe Creative Cloud Libraries lets you connect Creative Cloud Libraries with thousands of the most popular apps so you can automate your work and have more time for what matters most-no code required.
```

Comment:
@dches Your connector readme.md file is incomplete. Please follow all the instructions mentioned in the [documentation](https://github.com/microsoft/PowerPlatformConnectors#readmemd) and [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) is the template for readme.md.

Reply:
updated readme.md file